src/adapters/http/client.ts

```diff
@@ -0,0 +1,51 @@
+import HttpEventEmitter from "../../lib/httpEventEmitter.js"
+
+class HttpClientAdapter extends HttpEventEmitter {
+  async connect(): Promise<this> {
+    return this._connect()
+  }
+  async _connect(): Promise<this> {
+    // return new Promise(async (resolve, reject) => {
+      const requestObj = {
+        method: "POST",
+        url: "http://localhost:8081/trendingAnime",
+        body: {
+          name: "trendingAnime",
+          rating: 5,
+          studio: "teststudio",
+          genre: "testgenre",
+        },
+        query:{
+          name: "trendingAnime",
+          rating: "5",
+          studio: "teststudio",
+          genre: "testgenre",
+        }
+      }
+      this._send(requestObj);
+      return this
+    // });
+  }
+  async _send(message) {
```

Comment:
Try to make the Axios call in the _send function, use the server name and the channel name from he gleeMessage that you receive as a parameter in the `_send` function.